File: Python/Warmup-2/string_bits.py

# this function return a new string made of every other char starting with the first
def string_bits(str):
	s = ""

	for x in range(0,len(str),2):
		s = s + str[x]

	return s

print(string_bits("Hello"))
print(string_bits("Hi"))
print(string_bits("Heeololeo"))

File: JavaScript/Warmup-2/stringBits.js

// given a string, return a new string made of every other char starting with the first, so "Hello" yields "Hlo"
const stringBits = (str) => {

    let newStr = '';

    for (let i = 0; i < str.length; i = i + 2) {
        newStr = newStr + str[i];
    }

    return newStr;

};

console.log(stringBits('Hello'));
console.log(stringBits('Hi'));
console.log(stringBits('Heeololeo'));
